tools/pythonpkg/src/pyconnection.cpp

```diff
@@ -97,8 +97,7 @@ void DuckDBPyConnection::Initialize(py::handle &m) {
 	         "Serialize a query to protobuf on the JSON format", py::arg("query"))
 	    .def("get_table_names", &DuckDBPyConnection::GetTableNames, "Extract the required table names from a query",
 	         py::arg("query"))
-	    .def("__enter__", &DuckDBPyConnection::Enter, py::arg("database") = ":memory:", py::arg("read_only") = false,
-	         py::arg("config") = py::dict())
+	    .def("__enter__", &DuckDBPyConnection::Enter)
```

Comment:
Can't this call connect directly then?

Reply:
This is a method on the Connection class, so we already have a connection?

Reply:
Ahh, is because ```__enter__``` requires a ```DuckDBPyConnection &self```. Alright, looks good.